ggezchain/assetlist.json

```diff
@@ -0,0 +1,84 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "GGEZ1 Chain",
```

Comment:
this chain_name is not valid (cannot have space characters or capitalized letters), and must match the directory name exactly.